lib/cn/cn_metrics.h

```diff
@@ -6,6 +6,7 @@
 #ifndef HSE_KVDB_CN_METRICS_H
 #define HSE_KVDB_CN_METRICS_H
 
+#include <assert.h>
```

Comment:
Some combination of headers churn caused this to be necessary for the one assert statement in this header.